### PR TITLE
LOG-5907: add trace log level detection

### PR DESCRIPTION
**Pull request by vparfonov**
### Description

/cc @Clee2691 
/assign @jcantrill 

/cherry-pick 

### Links

- Depending on PR(s):
- Bugzilla:
- Github issue:
- JIRA: https://issues.redhat.com/browse/LOG-5907
- Enhancement proposal:

### CODE REVIEW

File: internal/generator/vector/filter/openshift/viaq/normalize.go [hunk 1]
```diff
@@ -24,6 +24,8 @@ if !exists(.level) {
     .level = "info"
   } else if match!(.message, r'^D[0-9]+|level=debug|Value:debug|"level":"debug"') {
     .level = "debug"
+  } else if match!(.message, r'^D[0-9]+|level=trace|Value:trace|"level":"trace"') {
```
Comment (Clee2691) [inline internal/generator/vector/filter/openshift/viaq/normalize.go:27]: I think the first regex should start with `T` for trace,
```suggestion
  } else if match!(.message, r'^T[0-9]+|level=trace|Value:trace|"level":"trace"') {
```

Comment (vparfonov) [reply internal/generator/vector/filter/openshift/viaq/normalize.go:27]: nice catch

**Pull request merged**